docs/ext/pages/index.rst

```diff
@@ -1,66 +1,264 @@
+            paginator = pages.Paginator(pages=self.get_pages(), custom_view=view)
+            await paginator.respond(ctx.interaction, ephemeral=False)
+
+        @pagetest.command(name="groups")
+        async def pagetest_groups(self, ctx: discord.ApplicationContext):
+            """Demonstrates using page groups to switch between different sets of pages."""
+            page_buttons = [
+                pages.PaginatorButton(
+                    "first", label="<<-", style=discord.ButtonStyle.green
+                ),
+                pages.PaginatorButton("prev", label="<-", style=discord.ButtonStyle.green),
+                pages.PaginatorButton(
+                    "page_indicator", style=discord.ButtonStyle.gray, disabled=True
+                ),
+                pages.PaginatorButton("next", label="->", style=discord.ButtonStyle.green),
+                pages.PaginatorButton("last", label="->>", style=discord.ButtonStyle.green),
+            ]
+            view = discord.ui.View()
+            view.add_item(discord.ui.Button(label="Test Button, Does Nothing", row=2))
+            view.add_item(
+                discord.ui.Select(
+                    placeholder="Test Select Menu, Does Nothing",
+                    options=[
+                        discord.SelectOption(
+                            label="Example Option",
+                            value="Example Value",
+                            description="This menu does nothing!",
+                        )
+                    ],
+                )
+            )
+            page_groups = [
+                pages.PageGroup(
+                    pages=self.get_pages(),
+                    label="Main Page Group",
+                    description="Main Pages for Main Things",
+                ),
+                pages.PageGroup(
+                    pages=[
+                        "Second Set of Pages, Page 1",
+                        "Second Set of Pages, Page 2",
+                        "Look, it's group 2, page 3!",
+                    ],
+                    label="Second Page Group",
+                    description="Secondary Pages for Secondary Things",
+                    custom_buttons=page_buttons,
+                    use_default_buttons=False,
+                    custom_view=view,
+                ),
+            ]
+            paginator = pages.Paginator(pages=page_groups, show_menu=True)
+            await paginator.respond(ctx.interaction, ephemeral=False)
+
+        @pagetest.command(name="update")
+        async def pagetest_update(self, ctx: discord.ApplicationContext):
+            """Demonstrates updating an existing paginator instance with different options."""
+            paginator = pages.Paginator(pages=self.get_pages(), show_disabled=False)
+            await paginator.respond(ctx.interaction)
+            await asyncio.sleep(3)
+            await paginator.update(
+                ctx.interaction, show_disabled=True, show_indicator=False
```

Comment:
should be `pages=self.get_pages(), show_disabled=True, show_indicator=False`

Reply:
The example isn't updating the page contents, only the Paginator options. Specifying a new set of pages is optional.

Reply:
Yes, but the `update` function has no `interaction` parameter. https://github.com/Pycord-Development/pycord/pull/629/files#diff-1f6ec18dba61aa8a11ec82e53015fe94a59672e8d32f38f8a545f0d825d67573R281-R295
This causes the example to fail.

Reply:
Ay, and sorry. I just saw that I did a comment on the doc. I wanted to comment the "update" command in the example :D https://github.com/Pycord-Development/pycord/pull/629/files/f4d0e0e5b9a8cf5c3e5247395b8a29731dd817ef#diff-cfd6351a3c919fdabef8f4e771e485dc9c1bd960697036520081b7cbbda6fd65R219

Reply:
Ah, I see the issue now. I'll fix the example shortly.